orc/src/test/java/org/apache/iceberg/orc/TestOrcDeleteWriters.java

```diff
@@ -73,7 +72,7 @@ public void createDeleteRecords() {
 
   @Test
   public void testEqualityDeleteWriter() throws IOException {
-    File deleteFile = temp.newFile();
+    File deleteFile = temp;
```

Comment:
we can probably remove `deleteFile` alltogether

Reply:
I've removed the deleteFile